src/stravalib/client.py

```diff
@@ -2035,6 +2035,10 @@ def _fill_buffer(self) -> None:
 
         entities = []
         for raw in raw_results:
+            # Note: this fails to validate if the start and end latlng are empty
```

Comment:
If they are empty lists. so this is a "run" that was super short and had gps data (you can see the map on straval) but no start/end latlon data propulated